docs/reference/metadata.md

```diff
@@ -175,54 +175,173 @@ Indentation options control aspects of the auto-indentation mechanism.
 `increaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be indented one level further.
+  lines after current line will be indented one level further.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>increaseIndentPattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*increase]]></string>
+  ```
+
+  Then, in the corresponding buffer view:
+
+  ```
+  line1
+  line2
+  increase
+    after increase1
+    after increase2
   ```
 
 `decreaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be unindented one level.
+  current line and lines after current line will be unindented one level.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>decreaseIndentPattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*decrease]]></string>
+  ```
+
+  Then, in the corresponding buffer view:
+
+  ```
+    line1
+    line2
+  decrease
+  after decrease1
+  after decrease2
   ```
 
 `bracketIndentNextLinePattern`
 : *Regex.*
   If it matches on the current line,
-  only the next line will be indented one level further.
+  only the next non-whitespace line will be indented one level further.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>bracketIndentNextLinePattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*bracket]]></string>
+  ```
+
+  Then, in the corresponding buffer view:
+
+  ```
+  line1
+  line2
+  line3
+  bracket
+    
+    first non-whitespace line after bracket
+  after bracket2
+  after bracket3
   ```
 
 `disableIndentNextLinePattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will not be indented further.
+  and `bracketIndentNextLinePattern`'s *Regex* also matches on the current line.
```

Comment:
Wouldn't bracketIndentNextLinePattern have matched on the previous line? 
https://forum.sublimetext.com/t/everything-you-n-ever-wanted-to-know-about-indentation-in-st3/26207?u=kingkeith